docs/walkthroughs/02-adding-event-handlers.md

```diff
@@ -6,6 +6,8 @@ What makes Slate great is how easy it is to customize. Just like other React com
 
 Let's use the `onKeyDown` handler to change the editor's content when we press a key.
 
+You can check a working version [here](https://codesandbox.io/s/slatejs-examples-adding-event-handlers-1nuv7).
```

Comment:
In general I would avoid linking to the word here as it reduces scannability of links in a page. I would either link to working version, or better I would link to a more specific title. The same feedback applies to other here links in this PR. Thanks!